tests/Parser/BitcoinMoneyParserTest.php

```diff
@@ -76,6 +76,8 @@ public function bitcoinExamples(): array
             ["\xC9\x8399.", 9900],
             ["\xC9\x830", '0'],
             ["\xC9\x830.020000000", 2],
+            ["\xC9\x830000.020000000", 2],
+            ["-\xC9\x830000.020000000", -2],
```

Comment:
Just to be sure that we don't wreak havoc with a `ltrim` in the future:

```suggestion
            ["-\xC9\x830000.020000000", -2],
            ["-\xC9\x83000000", 0],
```